File: cpp/4/10807.cpp
#include <iostream>

using namespace std;

int main() {
    int result = 0;
    int t, x;
    cin >> t;
    int arr[t];

    for (int i = 0; i < t; i++) {
        cin >> arr[i];
    }

    cin >> x;

    for (int i = 0; i < t; i++) {
        if (arr[i] == x) result++;
    }

    cout << result;

    return 0;
}

File: kotlin/4/10807.kt
fun main() {
    var t = readLine()!!.toInt()
    var arr = readLine()!!.split(" ").map { it.toInt() }
    var x = readLine()!!.toInt()
    var result = 0

    for (i: Int in arr) {
        if (i == x) {
            result++
        }
    }

    print(result)
}